runtime/common/src/weights/cumulus_pallet_xcmp_queue.rs

```diff
@@ -16,10 +16,10 @@
 //! Autogenerated weights for `cumulus_pallet_xcmp_queue`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-08-18, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2023-10-30, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
 //! HOSTNAME: `ip-10-0-0-176`, CPU: `Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz`
-//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("moonbase-dev"), DB CACHE: 1024
+//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("moonbase-dev")`, DB CACHE: 1024
```

Comment:
I see it was using EXECUTION: None before, I suspect it was \wasm` and not `native` @notlesh right ?

Reply:
Interesting, I think this is because the CLI options are ignored for wasm-execution now. But it hasn't made a difference in my testing...